package.json

```diff
@@ -94,6 +95,7 @@
     "start:prod": {
       "command": "node bin/prod-server",
       "env": {
+        "NODE_ENV": "production",
```

Comment:
Here too, I set these so that the PREFIX_URL gets correctly applied when Webpack creates links to the bundles